Dockerfile

```diff
@@ -28,4 +27,15 @@ RUN go install \
   -ldflags "-s -w -X main.version=$NFD_VERSION" \
   github.com/kubernetes-incubator/node-feature-discovery
 
-ENTRYPOINT ["/go/bin/node-feature-discovery"]
+RUN go test .
+
+
+# Create production image for funning node feature discovery
```

Comment:
`s/funning/running`

Reply:
Well spotted, thanks! Fixed in the latest version of this PR